init.lua

```diff
@@ -480,6 +480,12 @@ minetest.register_node("anvil:anvil", {
 	on_blast = function(pos, intensity)
 		local drops = {"anvil:anvil"}
 		local meta = minetest.get_meta(pos)
+		if meta:get_int("shared") == 1 then
+			drops[1] = ItemStack({
+				name = "anvil:anvil",
+				meta = {shared = 1, description = S("Shared anvil")}
```

Comment:
Where is the `description` field needed? The node properly sets its `"description"` metadata field in `after_place_node`.